reana_workflow_engine_serial/version.py

```diff
@@ -14,4 +14,4 @@
 
 from __future__ import absolute_import, print_function
 
-__version__ = "0.4.0"
+__version__ = "0.5.0.dev20190219"
```

Comment:
Please use `0.5.0.dev20190220` so that we have the same version as in `r-commons`.